topics/appendix/tools/intellij_platform_gradle_plugin/tools_intellij_platform_gradle_plugin_jetbrains_runtime.md

```diff
@@ -0,0 +1,126 @@
+<!-- Copyright 2000-2024 JetBrains s.r.o. and contributors. Use of this source code is governed by the Apache 2.0 license. -->
+
+# JetBrains Runtime (JBR)
+
+<link-summary>The IntelliJ Platform Gradle Plugin requires JetBrains Runtime (JBR), which can be provided in multiple ways.</link-summary>
+
+<include from="tools_intellij_platform_gradle_plugin.md" element-id="Beta_Status"/>
+<include from="tools_intellij_platform_gradle_plugin.md" element-id="faq"/>
+
+The IntelliJ Platform Gradle Plugin requires JetBrains Runtime (JBR) for running IDE locally, testing, or performing other actions that require Java Runtime adjusted by JetBrains.
+
+For the best experience, it is recommended to rely on JetBrains Runtime bundled with the IntelliJ Platform used for development.
```

Comment:
```suggestion
For the best experience, it is recommended to rely on JBR bundled with the IntelliJ Platform used for development.
```
Let's use "JBR" abbreviation everywhere after introducing it in the first paragraph.